android/capacitor/src/main/java/com/getcapacitor/Logger.java

```diff
@@ -21,7 +21,7 @@ public static void init(CapConfig config) {
     }
 
     private void loadConfig(CapConfig config) {
-        this.config = config;
+        Logger.config = config;
```

Comment:
@jcesarmobile is this change correct, or should `config` be an instance variable instead of a static one?